src/components/navbar.component.tsx

```diff
@@ -78,6 +78,22 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
               <Logo />
             </HeaderLink>
             <HeaderGlobalBar>
+              <ExtensionSlot extensionSlotName="add-patient-slot">
+                <HeaderGlobalAction
```

Comment:
This should be a registered extension from `@openmrs/esm-patient-registration-app`. Remove it here.